rollup.config.js

```diff
@@ -44,5 +44,5 @@ export default {
     terser(),
   ],
 
-  external: ['react', 'react-dom', ''],
+  external: ['react', 'react-dom', 'reakit', 'body-scroll-lock', ''],
```

Comment:
why we need this ?

Reply:
`body-scroll-lock` have unexported module, RollupJS  can't build Villain